source/includes/_deployments.md

```diff
@@ -268,5 +275,78 @@ Property | Type | Description
 
 
 
+### Response
+A `Deployment` resource.
+
+
+## Set Deployment Availability
+
+> Request Example
+
+```shell
+$ curl -X PATCH \
+    -H "Content-Type: application/json" \
+    -H "Tenant: <manager-tenant>" \
+    -u <manager-username>:<manager-password> \
+    -d '{"availability": "<availability>"}' \
+    "http://<manager-ip>/api/v3.1/deployments/<deployment-id>/set-availability"
+```
+
+```python
+# Python Client
+client.deployments.set_availability('<deployment-id>', '<availability>')
+```
+
+> Response Example
+
+```json
+{
+  "inputs": {
+    ...
+  },
+  "permalink": null,
+  "description": "deployment_1",
+  "blueprint_id": "blueprint_1",
+  "policy_types": {
+    ...
+  },
+  "tenant_name": "default_tenant",
+  "created_at": "2017-12-17T09:28:22.800Z",
+  "updated_at": "2017-12-17T09:29:20.750Z",
+  "created_by": "admin",
+  "policy_triggers": {
+    ...
+  },
+  "private_resource": false,
+  "resource_availability": "tenant",
+  "groups": {
+    ...
+  },
+  "workflows": {
+    ...
+  },
+  "id": "deployment_1",
+  "outputs": {
+    ...
+  }
+}
+
+```
+
+`PATCH "<manager-ip>/api/v3.1/deployments/{deployment-id}/set-availability"`
+
+Update the availability of the deployment.
+
+### URI Parameters
+* `deployment-id`: The id of the deployment to update.
+
+### Request Body
+
+Property | Type | Description
+--------- | ------- | -----------
+`availability` | string | Defines who can access the resource (required).
+
+The availability value can be only `tenant` because global deployment is not allowed.
```

Comment:
I recommend -> The availability value must be `tenant` because global availability is not allowed.